CHANGELOG.md

```diff
@@ -4,6 +4,25 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v3.0.0](https://github.com/voxpupuli/puppet-ghost/tree/v3.0.0) (2020-05-09)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-ghost/compare/v2.1.1...v3.0.0)
+
+**Breaking changes:**
+
+- drop Ubuntu support [\#96](https://github.com/voxpupuli/puppet-ghost/pull/96) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
The README still states `It is in beta development and tested on Ubuntu 12.04 and 14.04, loosely tested on CentOS.`

Reply:
thanks for the hint, I updated the README.md